### ISSUE TITLE
Bumping grunt dep in a patch release :(

### ISSUE THREAD

**Issue by olore** [rse/grunt-extend-config#4]
[Bumping](https://github.com/rse/grunt-extend-config/commit/b8b7c306b981ea9128cc05701eaf32d03ca3b8be) your dependency on grunt to ~1.0.1 in a patch release has caused us some pain.

Please replace the dependency on grunt back to 0.4, and bump at least the minor version.

```
npm ERR! node v0.12.13
npm ERR! npm  v2.15.0
npm ERR! code EPEERINVALID

npm ERR! peerinvalid The package grunt@0.4.5 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer grunt-compare-json@0.1.1 wants grunt@~0.4.1
npm ERR! peerinvalid Peer grunt-extend-config@0.9.4 wants grunt@~1.0.1
```

**Comment by RyanMG** [comment]
Its worth noting this repo's README still states `This plugin requires Grunt ~0.4.0`, AND http://gruntjs.com/ lists 1.0.1 as a development version, not as stable.

**Comment by rse** [comment]
Fixed in 0.9.5
